draft-ietf-radext-radiusdtls-bis.md

```diff
@@ -47,64 +47,90 @@ informative:
-The most important use of this specification lies in roaming environments where RADIUS packets need to be sent across insecure or untrusted networks.
-An example for a worldwide roaming environment that uses RADIUS over TLS to secure communication is eduroam as described in {{?RFC7593}}.
+The following terminology is used in this document:
 
-## Changes from RFC6614 (RADIUS/TLS) and RFC7360 (RADIUS/DTLS)
+RADIUS packet:
+: As defined in {{RFC2865}}.
 
-The following list contains the most important changes from the previous specifications in {{RFC6613}} (RADIUS/TCP), {{RFC6614}} (RADIUS/TLS) and {{RFC7360}} (RADIUS/DTLS).
+RADIUS packet type:
+: As defined in {{RFC2865}}.
 
-* {{?RFC6614}} referenced {{?RFC6613}} for TCP-related specification, RFC6613 on the other hand had some specification for RADIUS/TLS.
-  These specifications have been merged into this document, and therefore removes {{RFC6613}} as normative reference.
-* RFC6614 marked TLSv1.1 or later as mandatory, this specification requires TLSv1.2 as minimum and recommends usage of TLSv1.3.
-* RFC6614 allowed use of TLS compression, this document forbids it.
-* RFC6614 only requires support for the trust model "certificates with PKIX" ({{?RFC6614, Section 2.3}}). This document changes this. For servers, TLS-X.509-PKIX ({{tlsx509pkix}}, equivalent to "certificates with PKIX" in RFC6614) and TLS-PSK ({{tlspsk}}) is now mandated and clients must implement at least one of the two.
-* The mandatory-to-implement cipher suites are not referenced directly, this is replaced by a pointer to the TLS BCP.
-* The specification regarding steps for certificate verification has been updated.
-* {{RFC6613}} mandated the use of Status-Server as watchdog algorithm, {{?RFC7360}} only recommended it. This specification mandates the use of Status-Server for both RADIUS/TLS and RADIUS/DTLS.
-* {{RFC6613}} only included limited text around retransmissions, this document now gives more guidance on how to handle retransmissions, especially across different transports.
-* The rules for verifying the peer certificate have been updated to follow guidance provided in {{!RFC9525}}. Using the Common Name RDN for validation of server certificates is now forbidden.
-* The response to unwanted packets has changed. Nodes should now reply with a Protocol-Error packet, which is connection-specific and should not be proxied.
+RADIUS/TLS:
+: A RADIUS exchange transmitted using TLS over TCP.
 
-The rationales behind some of these changes are outlined in {{design_decisions}}.
+RADIUS/DTLS:
+: A RADIUS exchange transmitted using DTLS over UDP.
 
-# Conventions and Definitions
+RadSec:
+: A collective term for RADIUS/TLS and RADIUS/DTLS.
 
-{::boilerplate bcp14-tagged}
+RADIUS/UDP:
+: RADIUS transported over UDP as defined in {{RFC2865}}.
 
-Within this document we will use the following terms:
+(D)TLS handshake message:
+: As defined in DTLS {{RFC9147}}.
```

Comment:
Either it is truly TLS and DTLS, then we would need a reference to TLS as well, or its just DTLS, then remove the parenthesis.